glyphs-reader/src/font.rs

```diff
@@ -314,6 +316,51 @@ impl RawGlyph {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Hash)]
+pub struct Instance {
+    pub name: String,
+    pub active: bool,
+    pub type_: InstanceType,
```

Comment:
nit: we commonly spell this as `ty` in Rust land. Alternatively, `kind` and `InstanceKind`.

Reply:
I see `ty`, `typ` and `type_` in various places, happy to standardize on one or the other 🤷

Reply:
These names have to match the glyphs data and work with FromPlist, will add a comment